synthtool/sources/git.py

```diff
@@ -44,6 +44,33 @@ def make_repo_clone_url(repo: str) -> str:
         return f"https://github.com/{repo}.git"
 
 
+def _local_default_branch(path: pathlib.Path) -> str:
+    """Helper method to infer the default branch.
+
+    Sorts the list of branches by committerdate (latest is last) and then
+    returns the later of master or main. The order of branches that are tied
+    by committerdate is undefined.
+
+    Arguments:
+        path {pathlib.Path} - Path to the local git clone
+
+    Returns:
+        string -- The inferred default branch.
+    """
+    branches = (
+        subprocess.check_output(
+            ["git", "branch", "--sort=-committerdate", "--format=%(refname:short)"],
```

Comment:
Aha! That minus sign in front of commiterdate must mean descending order.  Cool.

Reply:
Yep, much easier than iterating the list in reverse.